scripts/sweep_layers.sh

```
@@ -0,0 +1,21 @@
+
+set -euo pipefail
+
+for N_LAYERS in 6 12 16 22; do
+    echo "Submitting job with lr=$N_LAYERS"
+    sbatch --job-name="eq_layers_${N_LAYERS}" \
+        scripts/train_eq.sh \
+        lightning_module.model.num_hidden_layers=$N_LAYERS \
+        only_preprocess=false
+
+    # If this is NOT the last job, wait 30 seconds
+    if [ $((i + 1)) -lt $NUM_LAYERS ]; then
```

Shellcheck (0.11.0)

[warning] i is referenced but not assigned.

(SC2154)